### ISSUE TITLE
Need to introduce downvote option to "best practice" and "clever" solution buttons

### ISSUE THREAD

**Issue by fxrbfg** [Codewars/codewars.com#1063]
In almost every cata many solutions marked as "best practice" but some of them actually tricky or has really awfull code that not according to best practice vote. After first vote them pop up in the top and other people continue to mark them as "good solutions" misleading novice to reproduce that in their code. It will be good to give community to downvote bad solutions like in stackoverflow and other sites.

**Comment by Voileexperiments** [comment]
Unfortunately, just like every other site CodeWars users mostly consists of noobs, so don't expect everyone to be able to vote for the best :(

You can still leave comments on a kata solution though. (Assuming it's not already flooded with... even more noobs asking about some basic syntax questions.)

**Comment by docgunthrop** [comment]
@fxrbfg Introducing a downvote button would likely exacerbate the situation by opening a new channel for abuse. The best course of action is to simply make a comment to the given solution explaining how it is not best practice / clever.

**Comment by andreAbtahi96** [comment]
Yeah, I definitely agree! As a beginner myself in codewars, I don't know how constructive a downvote would be if there is not any feedback.

**Comment by hobovsky** [comment]
My response originaly posted as an answer in a [duplicate discussion](https://github.com/codewars/codewars.com/discussions/3058):

> I think that the downvote button does not solve the problem of bad, upvoted solutions. The "bad" upvotes are there because users who upvote "bad" solutions simply fail to see what is bad about them: they don;t know better syntax, or they don't know that the solution has terrible performance characteristics, or they didn't get past the stage of "one-liners are awesome" yet. Similarly, the "downvote" button would be affected by the same problem: "good" solutions would be downvoted by users who aren't able to tell why a solution is good. This way, many good solutions with good performance or which use proper built-ins instead reinventing the wheel, will end up downvoted as ["too mysterious"](https://www.codewars.com/kata/reviews/541c8b5e7e4b4c61e2000149/groups/541df7aa259d9c7b80000a88#6502d2e030881a1ae3a2d682), or as [not in spirit of the game](https://www.codewars.com/kata/541a354c39c5efa5fa001372/solutions/python#622fc17acdd705000e30f227).
